lib/reducers/ui.js

```diff
@@ -38,6 +38,7 @@ const initial = Immutable({
   cursorBlink: false,
   borderColor: '#333',
   fontSize: 12,
+  fontWeight: 500,
```

Comment:
why setting `500` when you're setting `600` in config-default? 

also what is the default look like?